webpack.renderer.config.js

```diff
@@ -0,0 +1,58 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+const plugins = [
+  new HtmlWebpackPlugin({
+    template: path.resolve(__dirname, "src", "html", "index.html")

```

Comment:
So this is a lodash template now that `html-webpack-plugin` adds the webpack build script to?

Reply:
Not quite, [html-webpack-plugin](https://github.com/jantimon/html-webpack-plugin) is a way to make a generic template to add js into. It has a bit of it's own dsl to modify.  It is nice to put it into the output folder too